src/components/app/MenuButtons/index.js

```diff
@@ -98,10 +103,10 @@ class MenuButtonsImpl extends React.PureComponent<Props, State> {
       case 'from-addon':
       case 'unpublished':
       case 'from-file':
+      case 'local':
         return 'local';
```

Comment:
My first implementation was moving to the `'local'` datasource instead. I changed that but I thought that this change still made sense so I kept this small change nonetheless.